configure.ac

```diff
@@ -295,9 +295,6 @@ AC_MSG_CHECKING([what semaphore type to use]);
 AS_IF([test "x$have_mach" = "xtrue"],
   [AC_DEFINE(USE_MACH_SEM, 1, [Define to use Mach semaphores])
     AC_MSG_RESULT([Mach semaphores])],
-  [test "x$have_futex" = "xtrue"],
```

Comment:
that is very good, you should also get rid of all the code guarded with `#if USE_FUTEX_SEM` that is still around